### ISSUE TITLE
schema text isn't described [phab:T220]

### ISSUE THREAD

**Issue by kythe** [google/kythe#1759]
_Created by **pludemann** at 2017-02-22 21:53:50_: 

Schema doc node has this, but nowhere is `text` defined (`text` seems to be used for things other than `doc`, such as `lookup`).

```
//- DocNode.node/kind doc
//- DocNode.text " A function.\n It sums its parameters `[x]` and `[y]`."
```

**Comment by kythe** [comment]
_**fromberger** wrote at 2017-02-24 02:59:53_:

I think this probably falls under the more general task of documenting the marked-source message. Recategorizing on that basis.

**Comment by kythe** [comment]
revision #1829

**Comment by creachadair** [comment]
Fixed by #1829.
